src/logic/map_objects/walkingdir.cc

```diff
@@ -84,4 +84,23 @@ WalkingDir get_backward_dir(WalkingDir dir) {
 	}
 	NEVER_HERE();
 }
+
+std::string walkingdir_to_string(WalkingDir d) {
```

Comment:
Could you also implement the reverse and use it in soldier.cc and lua_map.cc? It's a bit out of scope, but this way it will be clean.